examples/apps/generic_client/proxy_mode_generic_client.py

```diff
@@ -0,0 +1,377 @@
+                    worker_id = w_id
+                    break
+            return worker_id
+        else:
+            logger.error("ERROR: Failed to lookup worker")
+            sys.exit(-1)
+
+    def create_work_order_params(self, worker_id, worker_encrypt_key,
+                                 session_key, session_iv,
+                                 verification_key):
+        # Convert workloadId to hex
+        workload_id = self._workload_id.encode("UTF-8").hex()
+        self.work_order_id = secrets.token_hex(32)
+        self.session_key = session_key
+        self.session_iv = session_iv
+        self.verification_key = verification_key
+        requester_id = secrets.token_hex(32)
+        requester_nonce = secrets.token_hex(16)
+        # Create work order params
+        try:
+            wo_params = WorkOrderParams(
+                self.work_order_id, worker_id, workload_id, requester_id,
+                session_key, session_iv, requester_nonce,
+                result_uri=" ", notify_uri=" ",
+                worker_encryption_key=worker_encrypt_key,
+                data_encryption_algorithm="AES-GCM-256"
+            )
+        except Exception as err:
+            return False, err
+        # Add worker input data
+        for value in self._in_data:
+            wo_params.add_in_data(value)
+
+        # Encrypt work order request hash
+        code, out_json = wo_params.add_encrypted_request_hash()
+        if not code:
+            return code, out_json
+
+        return True, wo_params
+
+    def verify_wo_res_signature(self, work_order_res,
+                                worker_verification_key,
+                                requester_nonce):
+        # Verify work order result signature
+        sig_obj = signature.ClientSignature()
+        status = sig_obj.verify_signature(
+            work_order_res, worker_verification_key,
+            requester_nonce)
+        if status == SignatureStatus.PASSED:
+            logger.info("Signature verification Successful")
+        else:
+            logger.error("Signature verification Failed")
+            return False
+
+        return True
+
+    def get_work_order_result(self, work_order_id):
+        # Get the work order result for direct/proxy model
+        res = self._work_order_instance.work_order_get_result(
+            work_order_id
+        )
+        logger.info("Work order result {}".format(res))
+        return res
+
+    def get_worker_details(self, worker_id):
+        # get the worker details for direct/proxy model.
+        # Retrieve worker details
+        status, _, _, _, details = self._worker_instance.worker_retrieve(
+            worker_id
+        )
+        logger.info("\n Worker status {} details : {}\n".format(
+            status, details
+        ))
+        if status == WorkerStatus.ACTIVE.value:
+            # Initializing Worker Object
+            worker_obj1 = worker_details.SGXWorkerDetails()
```

Comment:
Can we rename the variable to worker_obj since we don't have another instance of SGXWorkerDetails?